module7_selenium/src/test/java/bring_It_on/pages/PastebinResultPage.java

```diff
@@ -0,0 +1,33 @@
+package bring_It_on.pages;
```

Comment:
You should not create page object copies in different packages, just reuse them in different tasks.